recipes/bmi-wavewatch3/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "bmi-wavewatch3" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/csdms/bmi-wavewatch3/archive/v{{ version }}.tar.gz
+  sha256: 1ee3de05858cd76ef794b3594516c33355f503eddc31e8dcca836b6f6bbbd971
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+  entry_points:
+    - ww3 = bmi_wavewatch3.cli:ww3
+
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - bmipy
+    - cfgrib
+    - click
+    - tomli
+    - tqdm
+    - xarray
```

Comment:
Only packages needed by the setup.py should be in host.

Reply:
@beckermr Thanks. I've set host requirements to *python*, *pip*, and *setuptools*. Are *pip* and *setuptools* redundant? Should I just include *python*?